packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -228,7 +238,7 @@ export interface ResultConfiguration {
    *
    * @default - Query Result Location set in Athena settings for this workgroup
   */
-  readonly outputLocation?: s3.Location;
+  readonly outputLocation?: s3.Location | string;
```

Comment:
Unfortunately using a union type here will make any other language but typescript very unusable. They just turn into an `any` equivalent.

I see you have encountered an issue and I appreciate your drive to just go an fix it! In this case however, would you mind open an issue first? This gives our triage and support team a chance to have a look at it, possibly come with a workaround and at least inform the solution better.